<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractDialog</class>
 <widget class="QDialog" name="ExtractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract Potential</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../SB_Images.qrc">
    <normaloff>:/menu/sb2</normaloff>:/menu/sb2</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="extractLabel">
     <property name="text">
      <string>Extract (DBCG)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="extractSpin"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="moistureLabel">
     <property name="text">
      <string>Moisture (%)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="moistureSpin"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ppppgLabel">
     <property name="text">
      <string>PPPPG</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="ppppgValue">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="platoLabel">
     <property name="text">
      <string>Plato</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="platoValue">
     <property name="text">
      <string>pValue</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../SB_Images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExtractDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExtractDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
